frontend/views/utils/markdown-utils.js

```diff
@@ -14,24 +19,21 @@ marked.use({
 })
 
 export function renderMarkdown (str: string): any {
-  str = str.replace(/\n/g, '<br>') // firstly, manually replace all new-lines with <br>. ("breaks: true" option below doesn't consistently work.)
-    .replace(/<br>-/g, '\n-')
-
-  let converted = marked.parse(str, { gfm: true, breaks: true })
-
-  // remove unecessary line-breaks from the converted markdown outcome.
-  converted = converted.replace(/^\s+|\s+$/g, '')
+  // STEP 1. Handle multiple line-breaks
+  // markedjs with the gfm(Github Flavored Markdown) style always collapses multiple line-breaks into one
+  // so we need some custom logic to handle it manually.
+  // (Reference issue here: https://github.com/markedjs/marked/issues/190)
+  str = str.replace(/\n(?=\n)/g, '\n<br>')
+    .replace(/<br>\n(\s*)(\d+\.|-|```)/g, '\n\n$1$2')
+    .replace(/(\d+\.|-)(\s.+)\n<br>/g, '$1$2\n\n')
+  console.log('!@# str converted: ', str)
```

Comment:
Don't forget to remove extraneous console.log debug statements :)

Reply:
done